pkg/logs/input/kubernetes/launcher.go

```diff
@@ -35,6 +35,8 @@ type Launcher struct {
 	dockerRemovedServices     chan *service.Service
 	containerdAddedServices   chan *service.Service
 	containerdRemovedServices chan *service.Service
+	crioAddedServices         chan *service.Service
+	crioRemovedServices       chan *service.Service
```

Comment:
Ideally there should only be only `kubeletAddedService` & `kubeletRemovedService` so that we can support all kubernetes runtimes